tstest/integration/vms/gen/test_codegen.go

```diff
@@ -1,51 +0,0 @@
-// Copyright (c) Tailscale Inc & AUTHORS
```

Comment:
unrelated?

Reply:
I went to add a package doc to it and then found it was unused, as far as I could tell, so I fixed the package doc by deleting it.